LogTarget.php

```diff
@@ -165,6 +165,21 @@ protected function collectSummary()
     }
 
     /**
+     * Returns request time in seconds
+     *
+     * @return float request time in seconds.
+     * @see http://php.net/manual/en/function.microtime.php#example-2550
+     * @since 2.0.8
+     */
+    protected function getRequestTime()
```

Comment:
Using `protected` for the getter is inconssistent.
Either method should be public with intrernal field to store data:

```php
public function getRequestTime()
{
     if ($this->_requestTime === null) {
         ...
     }
     return $this->_requestTime;
}
```

or method should be renamed to get rid of `get` prefix, e.g. `determineRequestTime()`.